### PR TITLE
PHP8 compatibility updates

### PR DESCRIPTION
**Pull request by markbiek**
## Proposed changes:
These are minor changes as part of our PHP 8.1 compatibility update initiative pdqkMK-UN-p

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:
These are minor enough that I think visual inspection should be sufficient but please let me know if you need more information.

### CODE REVIEW

File: projects/packages/connection/src/class-error-handler.php [hunk 1]
```diff
@@ -277,7 +277,8 @@ public function store_error( \WP_Error $error ) {
 		$stored_errors[ $error_code ][ $user_id ] = $error_array;
 
 		// Let's store a maximum of 5 different user ids for each error code.
-		if ( count( $stored_errors[ $error_code ] ) > 5 ) {
+		$error_code_count = is_countable( $stored_errors[ $error_code ] ) ? count( $stored_errors[ $error_code ] ) : 0;
+		if ( count( $error_code_count ) > 5 ) {
```
Comment (nelsonec87) [inline projects/packages/connection/src/class-error-handler.php]: Wouldn't it be `if( $error_code_count > 5 ) { ` ?

**Review by nelsonec87 — CHANGES_REQUESTED** [review]

File: projects/packages/connection/src/class-error-handler.php (hunk 1 above)
Comment (markbiek) [reply projects/packages/connection/src/class-error-handler.php]: Yes, it would. Good catch! 🤦

**Review by nelsonec87 — DISMISSED** [review]
LGTM

**Review by nelsonec87 — APPROVED** [review]
LGTM

**Pull request merged**